src/baking_auth.c

```diff
@@ -230,33 +226,23 @@ bool parse_block(buffer_t *buf, parsed_baking_data_t *const out) {
  *   + (4 bytes)  uint32:  round of the related block
  *   + (32 bytes) uint8 *: hash of the related block
  */
-bool parse_consensus_operation(buffer_t *buf, parsed_baking_data_t *const out) {
+bool parse_consensus_operation(buffer_t *buf,
+                               parsed_baking_data_t *const out,
+                               bool is_attestation) {
     uint8_t tag;
 
     if (!buffer_read_u32(buf, &out->chain_id.v, BE) ||   // chain id
         !buffer_seek_cur(buf, 32u * sizeof(uint8_t)) ||  // ignore branch
-        !buffer_read_u8(buf, &tag) ||                    // tag
+        !buffer_read_u8(buf, &tag) ||                    // ignore tag
```

Comment:
ignore the `tag` using `!buffer_seek_cur(buf, sizeof(uint8_t))`

Reply:
Done.